sdk/communication/Azure.Communication.ProgrammableConnectivity/test-resources.json

```diff
@@ -0,0 +1,85 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentProgrammableConnectivity.json#",
```

Comment:
This needs to be updated.   Anyone should be able to run your tests in Live mode or record them using the Test Resources system - including the nightly/weekly pipelines.